src/main/java/cohen/weather/CurrentWeatherFrame.java

```diff
@@ -0,0 +1,94 @@
+package cohen.weather;
+
+import cohen.weather.jsonForFiveDay.FiveDayForecast;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+
+public class CurrentWeatherFrame extends JFrame
+{
+    private FiveDayForecast fiveDayForecast;
```

Comment:
Why is this a member variable?

Reply:
Not sure how to fix the issues caused by moving this into the constructor

Reply:
What issues?